package/boot/uboot-mediatek/patches/436-add-glinet-mt6000.patch

```diff
@@ -0,0 +1,250 @@
+--- /dev/null
++++ b/arch/arm/dts/mt7986a-glinet-gl-mt6000.dts
+@@ -0,0 +1,141 @@
++// SPDX-License-Identifier: GPL-2.0
++
++/dts-v1/;
++#include <dt-bindings/input/linux-event-codes.h>
++#include <dt-bindings/gpio/gpio.h>
++
++#include "mt7986.dtsi"
++
++/ {
++	#address-cells = <1>;
++	#size-cells = <1>;
++	model = "GL.iNet GL-MT6000";
++	compatible = "mediatek,mt7986", "mediatek,mt7986-emmc-rfb";
```

Comment:
Should be `glinet,gl-mt6000` as first compatible string (most specific) with the others following in the order from most specific to least specific (ie. `mediatek,mt7986` goes last)